client/NetPacksLobbyClient.cpp

```diff
@@ -133,6 +141,9 @@ bool LobbyUpdateState::applyOnLobbyHandler(CServerHandler * handler)
 
 void LobbyUpdateState::applyOnLobbyScreen(CLobbyScreen * lobby, CServerHandler * handler)
 {
+	if(!lobby) //stub: ignore message for game mode
```

Comment:
Does it mean you want to fix it later or is it kind of hack?

Reply:
It means, that there is no lobby screen during gameplay, so message shall be silently ignored